src/main/java/com/mathworks/ci/configuration/MatlabTestTaskConfigurator.java

```diff
@@ -42,6 +42,9 @@ public Map<String, String> generateTaskConfigMap(@NotNull final ActionParameters
         config.put(MatlabBuilderConstants.JUNIT_RESULTS_CHX, String.valueOf(params.getBoolean(MatlabBuilderConstants.JUNIT_RESULTS_CHX)));
         config.put(MatlabBuilderConstants.JUNIT_FILE, params.getString(MatlabBuilderConstants.JUNIT_FILE));
 
+        config.put(MatlabBuilderConstants.HTML_RESULTS_CHX, String.valueOf(params.getBoolean(MatlabBuilderConstants.HTML_RESULTS_CHX)));
+        config.put(MatlabBuilderConstants.HTML_FILE, params.getString(MatlabBuilderConstants.HTML_FILE));
```

Comment:
For HTML Test Report do User need to specify a folder path or file path ? If it is folder then we might want to name this appropriately.